src/api/partitioned-dbs.rst

```diff
@@ -194,3 +194,38 @@ See the guide for
           ],
           "total_rows": 4
         }
+.. _api/partitioned/find:
+
+``/db/_partition/partition_id/_find``
+=====================================
+
+.. http:get:: /{db}/_partition/{partition_id}/_find
+    :synopsis: Find a partition query by ID
```

Comment:
This description seems a bit unclear to me.  Is this finding a _partition query_ (if so, what is that?), or is it executing a find query only on a specific partition?